exporter/coralogixexporter/config.go

```diff
@@ -44,8 +47,34 @@ type Config struct {
 }
 
 func (c *Config) Validate() error {
-	if c.Endpoint == "" || c.PrivateKey == "" || c.AppName == "" || c.SubSystem == "" {
+	// check env variables
+	endpoint := c.Endpoint
+	privateKey := c.PrivateKey
+	appName := c.AppName
+	subSystem := c.SubSystem
+
+	if os.Getenv("CORALOGIX_ENDPOINT") != "" {
+		endpoint = os.Getenv("CORALOGIX_ENDPOINT")
+	}
+	if os.Getenv("CORALOGIX_PRIVATE_KEY") != "" {
+		privateKey = os.Getenv("CORALOGIX_PRIVATE_KEY")
+	}
+	if os.Getenv("CORALOGIX_APPLICATION_NAME") != "" {
+		appName = os.Getenv("CORALOGIX_APPLICATION_NAME")
+	}
+	if os.Getenv("CORALOGIX_SUBSYSTEM_NAME") != "" {
+		subSystem = os.Getenv("CORALOGIX_SUBSYSTEM_NAME")
+	}
+	if endpoint == "" || privateKey == "" || appName == "" || subSystem == "" {
 		return fmt.Errorf("One of your configureation field (endpoint, private_key, application_name, subsystem_name) is empty, please fix the configuration file")
 	}
+	c.Endpoint = endpoint
+	c.PrivateKey = privateKey
+	c.AppName = appName
+	c.SubSystem = subSystem
+
+	c.GRPCClientSettings.Endpoint = c.Endpoint
+	c.GRPCClientSettings.Headers = map[string]string{"ACCESS_TOKEN": c.PrivateKey, "appName": c.AppName, "subsystemName": c.SubSystem}
```

Comment:
What happens if users have specified headers themselves? You'd be overriding them here.

Reply:
Yes, I want to control about request header, I want to override the users.

Reply:
If your users are specifying any other HTTP headers, perhaps because a hop between them and you requires it, you'd be dropping their values without warning. Be nice to your users, if you really need to drop the headers they explicitly added, at least tell them you are removing it.

Reply:
Thanks, I will not override any HTTP header, only the specific header keys